mindmeld/components/entity_resolver.py

```diff
@@ -42,60 +49,295 @@
     load_index,
     resolve_es_config_for_version,
 )
+from ._util import _is_module_available, _get_module_or_attr
+from .. import path
+from ..core import Entity
+from ..exceptions import EntityResolverConnectionError, EntityResolverError
 
 logger = logging.getLogger(__name__)
 
+DEFAULT_ENTITY_RESOLVER_MODEL_CONFIGS = {
+    "exact_match": {
+        'model_type': 'exact_match',
+        "model_settings": {
+            "augment_lower_case": False
+        }
+    },
+    "text_relevance": {
+        'model_type': 'text_relevance',
+        # 'phonetic_match_types': ["double_metaphone"],
+    },
+    "sbert_cosine_similarity": {
```

Comment:
I think it makes sense to provide the developer default configs for each of the top keys like `sbert_cosine_similarity `, while also allowing them to change a sub-key like `pretrained_name_or_abspath ` in `config.py`